webapp/src/types/types.ts

```diff
@@ -86,11 +86,34 @@ export interface StudyMetadataPatchDTO {
 export interface StudyOutput {
   name: string;
   type: string;
+  settings: {
```

Comment:
What is the purpose to add it?

Reply:
its the correct shape of this type `settings` was missing